lib/LaTeXML/Package/LaTeX.pool.ltxml

```diff
@@ -2206,7 +2206,7 @@ DefPrimitiveI('\operator@font', undef, undef,
 #======================================================================
 
 DefMacro('\@tabacckludge {}', '\csname\string#1\endcsname');
-
+DefMacro('\@argdef', '\newcommand'); # stopgap?
```

Comment:
Technically, newcommand is defined in terms of argdef, not the other way around. etoolobx uses argdef directly, and I was surprised to see a simple aliasing to (latexml's) newcommand had great results.